sdk/eventhub/Azure.Messaging.EventHubs/stress/stress-test-resources.bicep

```diff
@@ -0,0 +1,95 @@
+@description('The url suffix to use when creating storage connection strings.')
+param storageEndpointSuffix string = environment().suffixes.storage
+
+@description('The maximum duration, in minutes, that a single test is permitted to run before it is considered at-risk for being hung.')
+param perTestExecutionLimitMinutes string = '15'
+
+@description('The client OID to grant access to test resources.')
+param testApplicationOid string
+
+@description('The location of the resource group.')
+param location string = resourceGroup().location
+
+var eventHubsNamespace_var = 'eh-${resourceGroup().name}'
```

Comment:
If you only use the resource group name here, it will be a little simpler to use the auto-generated namespace URL in the network blocking config, but it should still work fine with `eh-` we'll just have to remember to add it in other places.